inst/data_scripts/update_ds.R

```diff
@@ -0,0 +1,100 @@
+# Update DS by adding DSDECOD=RANDOMIZED rows
+
+library(dplyr)
+library(tidyselect)
+library(labelled)
+library(admiral)
+library(metatools)
+library(admiraltest)
```

Comment:
Sorry, you need to revert it to `admiral.test`.

CRAN rejected renaming `admiral.test` to `admiraltest`.

Reply:
sorry @Gopi2345 for the back and forth - i just made this update on your behalf and committed so no action for you for this one